packages/edit-site/src/components/page-pages/index.js

```diff
@@ -66,6 +68,10 @@ export default function PagePages() {
 		totalPages,
 	} = useEntityRecords( 'postType', 'page', queryArgs );
 
+	const { records: authors } = useEntityRecords( 'root', 'user', {
```

Comment:
This sort of data we don't necessary need to have right away. Only when user goes to filter (or edit cells). They could be lazy loaded upon events (opening the filters menu, etc.). Leave the comment here as food for thought in follow-ups.

Reply:
In general this is a filter that we shouldn't pre load all records, not only for performance, but also about the REST API limits to 100.

Reply:
My comment is a bit stale, since the approach has changed: the filter is now always present in the UI (before it was only visible as part of the columns' actions). Does that change your thinking? What should we do here instead?